app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,56 @@
+
+<%= render 'partials/navigation' %>
+
+<h1><%= @recipe.name%></>
+
+<section id="recipe-details">
+<div class="details-card">
+<p>Preparation time: <%=@recipe.preparation_time %> </p>
+<p>Cookin time: <%=@recipe.cooking_time %> </p>
+<p>Description: <%=@recipe.description %> </p>
+<%= link_to "Add ingredients",  new_user_recipe_recipe_food_path(recipe_id: @recipe) %>
+<%= link_to "Generate Shopping List", "#"%>
+</div>
+<table class="recipe-ingredients">
+<th>
+      <p>
+      <strong>Food</strong>
+      </p>
+    </th>
+    <th>
+      <p>
+    <strong>Quantity</strong>
+      </p>
+    </th>
+    <th>
+      <p>
+      <strong>Value</strong>
+      </p>
+    </th>
+    <th>
+      <p>
+      <strong>Actions</strong>
+      </p>
+    </th>
+  <% @recipe_foods.each do |recipe_food| %>
+      <tr>
+      <td>
+      <%=  recipe_food.food.name %>
+      </td>
+      <td>
+      <%= recipe_food.quantity%>
+      </td>
+      <td>
+      $<%= recipe_food.food.price*recipe_food.quantity %>
+      </td>
+
+      <td>
+      <% if can? :manage, @recipe %>
+      <%= button_to "Delete" , user_recipe_path(current_user, @recipe), method: :destroy %>
+      <% end %>
+
+      </td>
+      </tr>
+  <% end %>
+</table>
```

Comment:
- [ ] As required for the project. kindly make sure the Recipe details page matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_food.png?raw=true). You can check the missing details from the wireframe and them(e.g.: Public toggle button, Modify button in the table,...)

- [ ] Also make sure you fix the bug on the **Delete** button from the table, if it is clicked it raises the error below. 
![Screenshot from 2022-11-19 16-24-19](https://user-images.githubusercontent.com/83042742/202855649-85c41591-307a-46bc-9464-6aabd8dcb3af.png)